test/system/broadcasts_test.rb

```diff
@@ -5,23 +5,23 @@ class BroadcastsTest < ApplicationSystemTestCase
     visit messages_path
 
     assert_text "Messages"
-    assert_appends_text "Message 1" do |text|
-      Message.new(record_id: 1, content: text).broadcast_append_to(:messages)
+    assert_broadcasts_text "Message 1" do |text|
+      Message.create(id: 1, content: text).broadcast_append_to(:messages)
```

Comment:
Something is going on in the test suite that prevents the database from being wiped between test runs. This means that state is leaking between tests. This test is failing in CI in spite of the fact that this `Message.create` call is the first and only `Message` record in the test.